src/main/java/rx/Single.java

```diff
@@ -1,50 +1,37 @@
 /**
  * Copyright 2015 Netflix, Inc.
- * 
+ *
```

Comment:
These space removals are still there. Are you using IntelliJ? I'm sure there is a way to disable the remove whitespace from line ends option.